1.3/Patches/z_add_chitchat_thoughts.xml

```diff
@@ -135,43 +157,63 @@
 					<!-- ===pretty=== -->
 						<li>r_logentry(INITIATOR_thoughtLabel==pretty environment,tag=ReactToThought_NeedBeauty_pretty)->[thought_needbeauty_pretty]</li>
 						<li>thought_needbeauty_pretty->This place looks nice.</li>
+						<li>thought_needbeauty_pretty->This is a [moderately_attractive_obj] place.</li>
 					<!-- ===beautiful=== -->
 						<li>r_logentry(INITIATOR_thoughtLabel==beautiful environment,tag=ReactToThought_NeedBeauty_beautiful)->[thought_needbeauty_beautiful]</li>
-						<li>thought_needbeauty_pretty->I really like this place. It's beautiful.</li>
+						<li>thought_needbeauty_pretty->I really like this place. It's [intensifier]beautiful.</li>
+						<li>thought_needbeauty_pretty->I like being here. It's [intensifier]beautiful.</li>
+						<li>thought_needbeauty_pretty->This place is [intensifier]beautiful.</li>
 					<!-- ===gorgeous=== -->
 						<li>r_logentry(INITIATOR_thoughtLabel==gorgeous environment,tag=ReactToThought_NeedBeauty_gorgeous)->[thought_needbeauty_gorgeous]</li>
 						<li>thought_needbeauty_gorgeous->This place is truly beautiful. It lifts me up and makes me love life!</li>
-						<li>thought_needbeauty_gorgeous->This place is a dream come true. I'm so happy to be here!</li>
+						<li>thought_needbeauty_gorgeous->This place is a dream come true. I'm [intensifier]happy to be here!</li>
 						<li>thought_needbeauty_gorgeous(INITIATOR_artistic_level>=5)->This place looks like a true masterwork of fine arts!</li>
 				<!-- ===NeedComfort=== -->
 					<!-- ===uncomfortable=== -->
 						<li>r_logentry(INITIATOR_thoughtLabel==uncomfortable,tag=ReactToThought_NeedComfort)->[thought_need_comfort1]</li>
 						<li>thought_need_comfort1->My back is aching. Our furniture here is quite lacking.</li>
+						<li>thought_need_comfort1->Ouch! I must have sat down wrong. I think I blame our furniture...</li>
+						<li>thought_need_comfort1->Our furniture isn't very good. I'm [intensifier]uncomfortable.</li>
+						<li>thought_need_comfort1(INITIATOR_trait==ascetic,priority=1)->Not to complain, but this furniture causes my body to ache.</li>
 					<!-- ===comfortable=== -->
 						<li>r_logentry(INITIATOR_thoughtLabel==comfortable,tag=ReactToThought_NeedComfort)->[thought_need_comfort2]</li>
 						<li>thought_need_comfort2->I'm glad we have comfortable furniture around here.</li>
+						<li>thought_need_comfort2->Our furniture is fine. I'm comfortable enough!</li>
+						<li>thought_need_comfort2->This furniture is just what I needed.</li>
 					<!-- ===quite comfortable=== -->
 						<li>r_logentry(INITIATOR_thoughtLabel==quite comfortable,tag=ReactToThought_NeedComfort)->[thought_need_comfort3]</li>
 						<li>thought_need_comfort3->I'm so glad we have nice and comfortable furniture around here!</li>
+						<li>thought_need_comfort3->I'm comfortable here, and it feels pretty good.</li>
+						<li>thought_need_comfort3->I like this furniture. It's [intensifier]nice and comfortable!</li>
```

Comment:
"This" is risky because the pawn might comment this while being in the fields or wilderness...